### ISSUE TITLE
[BLOCKED][Settings][Desktop] Page is modal but also full screen with some elements entered in full width and others not

### ISSUE THREAD

**Issue by NVLudwig** [unoplatform/uno.todo#163]
## Current behavior

Page is modal but also full screen with some elements entered in full width and others left aligned in narrow column: Dividers are narrow, Version number is wide
![image](https://user-images.githubusercontent.com/29584087/165970342-5cd4264a-a89d-4c1f-9153-f00c85e4546e.png)

## Expected behavior

A modal page on desktop should appear either over rest of the main page (main page dimmed) to emphasize the navigation OR have it in a pane coming out from the right edge with the rest os the page dimmed.

_Hack fix:_ Just fit the header and version number into the left narrow column. defined by the chips and dividers

## How to reproduce it (as minimally and precisely as possible)

Enter Settings in desktop mode on a wide screen. Observe.

## Environment

MacOS 
Chrome Version 100.0.4896.75 (Official Build) (x86_64)
 Private window

Nuget Package:

Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [X] WebAssembly
- [ ] WebAssembly renders for Xamarin.Forms
- [ ] Windows
- [ ] Build tasks

Visual Studio:

- [ ] 2017 (version: )
- [ ] 2019 (version: )
- [ ] for Mac (version: )

Relevant plugins:

- [ ] Resharper (version: )

**Comment by agneszitte-nventive** [comment]
@nickrandolph / @Xiaoy312 we will need to discuss about this page.

**Comment by agneszitte-nventive** [comment]
Started to test some changes on this branch https://github.com/unoplatform/uno.todo/commits/dev/agzi/I163
On hold for now, as I am waiting on more guidance regarding the UX for the Settings page (Page ? Flyout ? Don't forget the SignOut Dialog that can be opened on top of it)
(cc @nickrandolph, @NVLudwig)

**Comment by agneszitte-nventive** [comment]
Settings flyout started to be integrated (cf. #334) but there is a known issue for the SignOut, we will need to unblock this issue to be able to move forward with this.
Regarding the discussion about SignOut MessageDialog opening on top of the Settings dismissable Flyout
(cc @jeromelaban, @nickrandolph, @dr1rrb)
